### ISSUE TITLE
The bug with nested modules is not fixed

### ISSUE THREAD

**Issue by iaroslaff** [edgedb/edgedb-ui#234]
The bug with **nested modules** not fixed due to build crash.
[https://github.com/edgedb/edgedb-ui/issues/230](url)

**Comment by jaclarke** [comment]
Yes, unfortunately the nested module fixes are not yet in any nightly or stable builds. It seems there were some changes to multiranges that was breaking schema introspection in the nightly ui, which I've just pushed a fix for (https://github.com/edgedb/edgedb-ui/pull/237). The nightly builds themselves are still failing, but that is being worked on, so hopefully there should be a working nightly with the nested module fixes soon. And, as far as I know, there should be a new stable 3.x release sometime in the next few weeks.

**Comment by iaroslaff** [comment]
Сan be closed, I checked, the fixes are already applied, types are displayed again :ok_hand:
